### PR TITLE
feat: Added temp range support

### PR DESCRIPTION
**Pull request by cdnninja**

### CODE REVIEW

**Comment by cdnninja** [comment]
Will merge to confirm this works for Canada.  If so will move the same logic into other regions.

**Comment by cdnninja** [comment]
@fuatakgun this is ready for your review with temp range support.  I have left USA as is.  It reports Low and High as something our old override but I suspect we shouldn't force that to a number since in many cars low and high are valid values.

File: hyundai_kia_connect_api/KiaUvoApiCA.py [hunk 1]
```diff
@@ -118,6 +119,15 @@ def get_cached_vehicle_status(self, token: Token):
 
         vehicle_status = {}
         vehicle_status["vehicleStatus"] = response
+        
+        #Coverts temp to usable number.  Currently only support celsius. Future to do is check unit in case the care itself is set to F. 
+        
+        value = vehicle_status["vehicleStatus"]["airTemp"]["value"]
```
Comment (fuatakgun) [inline hyundai_kia_connect_api/KiaUvoApiCA.py:125]: this looks like a perfect function into util.py eg;
```
get_hex_temp_into_index(value):
    value = value.replace("H", "")
    value = value.replace("C", "")
    value = "0x" + value
    value = int(value,16)
```

**Review by fuatakgun — CHANGES_REQUESTED** [review]
can we move shared stuff into a function?

rest looks good to me, thanks

File: hyundai_kia_connect_api/KiaUvoApiCA.py (hunk 1 above)
Comment (cdnninja) [reply hyundai_kia_connect_api/KiaUvoApiCA.py]: Done.

**Comment by cdnninja** [comment]
As a note reason I was naming c vs F is that these cars can return but units.  Depends what the car itself is set to.  We can update that later though if required.  Right now I don't know the range a Canadian car supports in F.  Not worth my time to play with.

**Comment by cdnninja** [comment]
Need your thoughts on how to fix this.  I removed imports on util since this causes circular imports.  This would cause the one function to fail though.   How do we best fix this?

**Review by fuatakgun — APPROVED** [review]

**Pull request merged**